docs/source/pages/tutorials/index.rst

```diff
@@ -275,3 +275,121 @@ dependency is manually uploaded to Maven Central and does not meet the security
 You can use this policy in your GitHub Actions to prevent a deployment or fail a CI test during the
 development. Alternatively, you can treat the result as a warning and manually investigate the
 dependencies to make sure they are secure and can be trusted.
+
+-----------------------------------------------------------------
+More Accurate Analysis via Mappings of Artifacts to Exact Commits
+-----------------------------------------------------------------
+
+This tutorial demonstrates how Macaron can be used to determine the differences between one or more states of the single open source repository that produced one or more related artifacts. In this way, we show how a developer can be potentially mislead by supply chain security information that has been created for the current state of an artifact's source repository, rather than the version of the artifact they are actually using.
```

Comment:
```suggestion
This tutorial demonstrates how Macaron can be used to determine the differences between one or more states of the single open source repository that produced one or more related artifacts. In this way, we show how a developer can be potentially misled by supply chain security information that has been created for the current state of an artifact's source repository, rather than the version of the artifact they are actually using.
```